src/Language/PureScript/AST/Literals.hs

```diff
@@ -1,10 +1,15 @@
 -- |
 -- The core functional representation for literal values.
 --
-module Language.PureScript.AST.Literals where
+module Language.PureScript.AST.Literals 
+  ( module Language.PureScript.AST.Literals 
+  , NumericLiteral(..)
+  , foldNumericLiteral
+  ) where
 
 import Prelude.Compat
 import Language.PureScript.PSString (PSString)
+import Language.PureScript.Parser.Lexer (NumericLiteral(..), foldNumericLiteral)
```

Comment:
Putting the `NumericLiteral` type into the `Lexer` module seems weird to me, but I'm not sure a better place to locate it.

Reply:
`Language.PureScript.AST.Literals`? I think the module comment in `AST.Literals` is a bit misleading, as that `Literal` type appears to be used in at least one other place than the core functional representation (namely, the AST), so I think it makes sense to put it there.